ambry-router/src/main/java/com/github/ambry/router/BatchOperationCallbackTracker.java

```diff
@@ -42,13 +43,18 @@
    * @param callback the {@link Callback} to be triggered once acks are received for all blobs
    */
   BatchOperationCallbackTracker(List<BlobId> blobIds, FutureResult<Void> futureResult, Callback<Void> callback) {
+    this(blobIds, futureResult, callback, null);
+  }
+
+  BatchOperationCallbackTracker(List<BlobId> blobIds, FutureResult<Void> futureResult, Callback<Void> callback, Callback<Void> completionCallback) {
```

Comment:
why do we need another callback here, we already have `callback`, there is no need to have `completionCallback`.

Reply:
fixed.

Reply:
fixed.